Snowflake.Data/Core/ChunkDownloaderFactory.cs

```diff
@@ -31,7 +31,8 @@ class ChunkDownloaderFactory
                         responseData.chunks,
                         responseData.qrmk,
                         responseData.chunkHeaders,
-                        cancellationToken);
+                        cancellationToken,
+                        resultSet.sfStatement.SfSession.restRequester);

```

Comment:
Do we need to update other chunk downloaders as well?

Reply:
Yes, I did update them too. 
The other chunk downloaders have the resultset object and are getting the restRequester from there.
See changes in SFBlockingChunkDownloader.cs and SFBlockingChunkDownloaderV3.cs.